wqflask/wqflask/static/new/css/markdown.css

```diff
@@ -11,3 +14,54 @@
     margin-right: auto;
     margin-left: auto;
 }
+
+
+
+.github-btn {
+    margin: 20px 10px;
+    display: flex;
+    width: 90vw;
+    justify-content: flex-end;
+}
+
+.github-btn a {
+    display: inline-block;
+    align-self: center;
+    text-decoration: none;
+    transition-property: all;
+    transition-duration: 0.5s;
+    padding: 8px 12px;
+    color: #fff;
+    /*background: #27407a;*/
+    background: blue;
+    border-radius: 20px;
+
+    /*text-align: center;*/
+
+
+}
+
+.github-btn a:hover {
+
+    background: #333;
+
+}
+
+
+.container {
+    width: 80vw;
+    margin: auto;
+    max-width: 80vw;
+
+}
+
+.container p {
+    font-size: 17px;
+    word-spacing: 0.2em;
+}
+
+@media(max-width:650px) {
+    .container {
+        width: 100vw;
+    }
+}
```

Comment:
**nitpick:** Consider configuring your editor to have unix line endings

Please see(if you are using sublime): https://gist.github.com/whophil/06398a997e661349e0e3. It's a nice habit to get into. You never know when you'll cat 2 files together, and that line ending comes in handy there... Really a nitpick though :)

Reply:
Thanks this helped